docs/source/using.rst

```diff
@@ -504,9 +504,9 @@ simple table summary of profiling results is displayed::
      ncalls  tottime  percall  cumtime  percall filename:lineno(function)
           1    0.000    0.000    6.844    6.844 asv/benchmark.py:171(method_caller)
           1    0.000    0.000    6.844    6.844 asv/benchmark.py:197(run)
-          1    0.000    0.000    6.844    6.844 /usr/lib64/python2.7/timeit.py:201(repeat)
-          3    0.000    0.000    6.844    2.281 /usr/lib64/python2.7/timeit.py:178(timeit)
-          3    0.104    0.035    6.844    2.281 /usr/lib64/python2.7/timeit.py:96(inner)
```

Comment:
Just out of curiosity, are these the result of a rename or did you re-run the relevant snippets?

Reply:
These are all just the result of a manual rename